applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -144,7 +150,7 @@ static void on_bt_ready(int err)
 	ret = ble_hci_vsc_adv_tx_pwr_set(CONFIG_NRF_21540_MAIN_DBM);
 	ERR_CHK(ret);
 
-	LOG_DBG("TX power set to %d", CONFIG_NRF_21540_MAIN_DBM);
+	LOG_DBG("Advertisement TX power set to %d", CONFIG_NRF_21540_MAIN_DBM);
```

Comment:
Is this also advertisement?

Reply:
Yes this is also related to the ble_hci_vsc_adv_tx_pwr_set() function